### PR TITLE
feat: implement executor gui

### PR DESCRIPTION
**Pull request by ahkohd**
to run

```bash
cd executor-gui
pnpm i
pnpm tauri dev
```

**Screenshot:**
1-show

2-show

### Summary by CodeRabbit

- New Feature: Introduced a new executor GUI with various components for managing and controlling the executor.
- New Feature: Added an HTML entry point for the Tauri + React + TS application.
- New Feature: Implemented several contexts to manage executor control, logging, store access, and view state.
- New Feature: Provided a custom hook `useToastFormikErrors` for displaying Formik errors as toast notifications.
- New Feature: Added utility functions for retrieving configuration values and wrapping asynchronous function calls with toast notifications.
- Style: Integrated Tailwind CSS and added base styles for the application.
- Chore: Updated `.gitignore`, `.eslintignore`, and `.eslintrc` files, and added a PostCSS configuration file.

### CODE REVIEW

**Review by 3h4x — CHANGES_REQUESTED** [review]
very good stuff! 🚀 

please remove changes to Cargo from PR to not trigger runtime upgrade

File: Cargo.toml [hunk 1]
```diff
@@ -262,6 +262,7 @@ members = [
   "sdk/crates/sdk",
   "types",
   "types/abi",
+  "client/packages/executor-gui/src-tauri"
```
Comment (ahkohd) [inline Cargo.toml:265]: @3h4x if I remove this line, I won't be able to build the tauri project.

Comment (3h4x) [reply Cargo.toml:265]: this is no good then... 
how will CI know that this change is not our blockchain change?
with any change to tauri CI will want to do runtime upgrade. we need to solve it before merging.
I guess we need to move it to seperate repo then

**Comment by ahkohd** [comment]
Closed in favour of #1420 and [feat: executor-gui](https://github.com/t3rn/frontend/pull/123)

**Pull request closed without merging**